frontend/src/components/Contributions/ContributionForm.vue

```diff
@@ -88,6 +88,8 @@
   </div>
 </template>
 <script>
+const PATTERN_NON_DIGIT = /\D/g
```

Comment:
maybe use a standard for regex definitions when it comes to naming - how would that standard look like?

Reply:
This is bad code. I focused on the links. To validate the amount field, we need a proper solution, better than this regex. But this is another issue.